toolchain/base/value_store_types.h

```diff
@@ -13,16 +13,10 @@
 namespace Carbon {
 
 // Common calculation for ValueStore types.
-template <typename IdT, typename ValueT = IdT::ValueType,
-          typename KeyT = ValueT>
+template <typename IdT, typename ValueT = IdT::ValueType>
```

Comment:
Is this default template argument still being used for anything?

Reply:
Sent off #5761 to do the cleanup